### PR TITLE
Merge upstream prysm

### PR DESCRIPTION
**Pull request by Inphi**
Merged the latest develop branch on upstream.
Passing interop tests on [CI](https://github.com/Inphi/eip4844-interop/pull/26).

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Cool you got this to work! What was the trick? Merging from develop instead of the terence rebase?

**Comment by Inphi** [comment]
yup. Cleaner to merge from develop as Terence branch also contains the coupled Beacon Block code.
We can add that in later.

File: proto/prysm/v1alpha1/cloners.go [hunk 1]
```diff
@@ -512,3 +512,53 @@ func CopyBlindedBeaconBlockBodyBellatrix(body *BlindedBeaconBlockBodyBellatrix)
 		ExecutionPayloadHeader: CopyExecutionPayloadHeader(body.ExecutionPayloadHeader),
 	}
 }
+
+// CopySignedBeaconBlockWithBlobKZGs copies the provided SignedBeaconBlockBellatrix.
```
Comment (roberto-bayardo) [inline proto/prysm/v1alpha1/cloners.go]: Updated comment (Bellatrix -> WithBlobKZGs) here and for functions below it.

**Review by roberto-bayardo — APPROVED** [review]

File: proto/prysm/v1alpha1/cloners.go (hunk 1 above)
Comment (Inphi) [reply proto/prysm/v1alpha1/cloners.go]: Nice catch.

Comment (Inphi) [reply proto/prysm/v1alpha1/cloners.go]: fixed. Thanks!

**Pull request merged**